Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ColorPickerButton/ColorPickerButtonPage.xaml

```diff
@@ -0,0 +1,12 @@
+﻿<Page x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.ColorPickerButtonPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+      mc:Ignorable="d"
+      Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <Grid />
```

Comment:
We need a 'shadow copy' of the control here even though it's not used as a reference so it doesn't get optimized out by the .NET Native compiler for Release builds.

Reply:
> Think the biggest UX related issue I saw was around selection/value destruction compared to how the original one works. If we can solve that, I think that'd be really something!
I don't understand what you mean by this.

> We need a 'shadow copy' of the control here even though it's not used as a reference so it doesn't get optimized out by the .NET Native compiler for Release builds.
I also don't understand this. The .NET native is the cause of many production bugs but I don't know what you mean here -- you'll have to give an example.

Reply:
> Think the biggest UX related issue I saw was around selection/value destruction compared to how the original one works. If we can solve that, I think that'd be really something!

Was calling out the HSV/RGB issues below more prominently saying that after picking a Hue/Color in the picker, operations to modify the shade should still allow maintaining that color picked. This may be a gap in my understanding of the intent of the UI here, but as a user I felt like getting locked on black was a bad experience.

For shadow copy, see example here:

https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/a47305cbaea5cfeb8ecdfd8c74e2570b86ed26e2/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TokenizingTextBox/TokenizingTextBoxPage.xaml#L13-L17

Since we don't use the control directly in the sample it won't work in Release mode unless we do this.

Reply:
> Was calling out the HSV/RGB issues below more prominently saying that after picking a Hue/Color in the picker, operations to modify the shade should still allow maintaining that color picked. This may be a gap in my understanding of the intent of the UI here, but as a user I felt like getting locked on black was a bad experience.

You are saying two different things I think. 

 (1) Making sure the original color is available by pressing back/forth in the mini-palette.

I simplified the algorithm so this is technically possible but it breaks other things. Overall, this isn't the goal either. The goal is to generate similar colors but different shades -- like windows accent colors. It's really a palette of different colors not shades of a single color. With perceptual differences, even working in HSV space, it isn't possible to preserve the original selection unless a history of past selections is maintained. As it is, if the value channel is at max the algorithm breaks and won't generate alternate shades because the saturation and even hue would need to be adjusted. Again, I would like to use the windows algorithm which doesn't seem available. You might need to talk to someone internally and see if this is proprietary or can be shared. https://github.com/windows-toolkit/WindowsCommunityToolkit/issues/3363#issuecomment-650550731

 (2) Getting stuck at max (white) and min (black) colors. I specially handle these now so shades of gray will be generated to get you unstuck in the mini-palette. Color information is already lost though and the converter has no knowledge of past selections. Gray is the best compromise I can think of.